include/NovelRT/Graphics/Camera.h

```diff
@@ -1,14 +1,9 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
 
-#ifndef NOVELRT_NOVELCAMERA_H
-#define NOVELRT_NOVELCAMERA_H
-#include "GeoMatrix4.h"
-#include "Lazy.h"
-#include "NovelUtilities.h"
-#include "CameraViewChangedEventArgs.h"
-
-namespace NovelRT {
-  class NovelCamera {
+#ifndef NOVELRT_GRAPHICS_CAMERA_H
+#define NOVELRT_GRAPHICS_CAMERA_H
+namespace NovelRT::Graphics {
```

Comment:
nit: We consistently have a newline between the `#define` and `namespace` elsewhere